week-3-practise/exercise-1-Mohammad .js

```diff
@@ -0,0 +1,23 @@
+/* 
+  You are given an array of student marks.
+  If all marks are over 10 the student is graduated, otherwise should try again. 
+  Using .every() (and any other array methods necessary), find out the student is graduated or not.
+  If the student is graduate print "Congratulation, You are graduated", otherwise print "You are nearly there, try again".
+*/
+
+var marks = [11, 19, 13, 20, 18, 16, 10];
+function studentResult(arr) {
```

Comment:
Cool! I think the if/else can be improved :) Are you sure you need to compare `arr.every(studentMarks)` to `true` to get a boolean?